operations/research-and-development/engineering/integrations.md

```diff
@@ -89,6 +89,24 @@ Seeing an intermittent connection issue with Zoom on http://dkh-local.ngrok.io.
 - Staff members add the issues there are working on for the current sprint to https://github.com/orgs/mattermost/projects/1.
     - Putting an issue is always preferred over putting a PR on the board.
 
+## Documentation process
+
+**1% draft**
+
+This guide outlines the processes and workflows for requesting, updating, managing, and maintaining documentation for Mattermost-managed and community-managed apps, plugins, and integrations. This includes the branching strategy for the Gitbook repos, how to set up a new Gitbook, and the process for aligning documentation with releases.
+
+Integrations comprise Mattermost-managed plugins and community-managed integrations.
```

Comment:
I'm stumbling on the term "integrations". What does it refer to in this context?